fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingScrubDangling.java

```diff
@@ -179,7 +179,7 @@ public IndexingScrubDangling(@Nonnull final IndexingCommon common,
 
             return iterateRangeOnly(store, cursor,
                     this::deleteIndexIfDangling,
-                    lastResult, hasMore, recordsScanned)
+                    lastResult, hasMore, recordsScanned, true)
```

Comment:
To me it seems that if we'll add something like: 
``` isIdempotent = store.getIndexMaintainer(index).isIdempotent() ```
We'll also have to support it all over the place (as in IsolationLevel, etc). Since we can't, chances are that it will only add confusion to an already non-trivial code. Does it make sense?